relay-server/src/metrics_extraction/snapshots/relay_server__metrics_extraction__event__tests__extract_span_metrics_all_modules.snap

```diff
@@ -1692,4 +1692,52 @@ expression: metrics
             "span.op": "resource.script",
         },
     },
+    Bucket {
+        timestamp: UnixTimestamp(1702474613),
+        width: 0,
+        name: "d:spans/exclusive_time@millisecond",
+        value: Distribution(
+            [
+                32.000065,
+            ],
+        ),
+        tags: {
+            "environment": "fake_environment",
+            "span.description": "my-component-name",
```

Comment:
@0Calories the span description is overwritten by the component name in this implementation, because `span.description` on span metrics and in the spans dataset has so far always contained the human readable group name. This does mean that the original description is only available via nodestore though.